src/schema/paddle.ts

```diff
@@ -429,6 +463,24 @@ export const resolvers: IResolvers<unknown, AuthContext> = traceResolvers<
 
       return consolidated.filter(Boolean) as BasePricingPreview[];
     },
+    pricingPreviewByIds: async (
```

Comment:
Where is this going to be used?

Reply:
For the Funnel since they have a separate pricing IDs as compared to what is displayed from the Plus page.

Reply:
We can't use the method we have and then funnel on its side looks for pricing ids it needs? I am not really sure how it works but we are again starting to have one to many queries for pricing 😆